lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -141,6 +140,21 @@ def create_response_info
           }
         end
 
+        def all_passed?
+          transaction_status_passed? &&
+            true_id_product.present? &&
+            product_status_passed? &&
+            doc_auth_result_passed?
+        end
+
+        def attention_with_barcode?
+          return false unless doc_auth_result_attention?
+
+          parsed_alerts[:failed]&.count.to_i == 1 &&
+            parsed_alerts[:failed][0][:name] == '2D Barcode Read' &&
+            parsed_alerts[:failed][0][:result] == 'Attention'
```

Comment:
personally I like `.first` more than `[0]` but it's not a huge deal

```suggestion
            parsed_alerts[:failed].first[:name] == '2D Barcode Read' &&
            parsed_alerts[:failed].first[:result] == 'Attention'
```

but also on second thought, I think this would be a good `.dig` candidate:

```suggestion
            parsed_alerts.dig(:failed, 0, :name) == '2D Barcode Read' &&
            parsed_alerts.dig(:failed, 0, :result) == 'Attention'
```